packages/node_modules/@webex/internal-plugin-device/src/interceptors/device-url.js

```diff
@@ -24,18 +24,15 @@ export default class DeviceUrlInterceptor extends Interceptor {
    */
   onRequest(options) {
     const {headers, service, uri} = options;
-    const {devices, services} = this.webex.internal;
+    const {device, services} = this.webex.internal;
+    const cdu = 'cisco-device-url';
 
-    // Check if header is already set before moving forward
-    if (!devices.url || (headers && !headers['cisco-device-url'])) {
+    // Check if header is already set before moving forward.
+    if (!device.url || (headers && cdu in headers && !headers[cdu])) {
```

Comment:
is the `cdu in headers` check necessary? Since you're also checking for `!headers[cdu]`

Reply:
Yes, @taymoork2 and I discussed removing it, but after we merged the changes, other plugins would fail tests.

This is due to how `cdu` has to exist as a key, but return `false` when `!headers[cdu]` is interpreted in order for this interceptor's logic to flow properly.